packages/gatsby-source-wordpress/__tests__/process-node.test.js

```diff
@@ -43,3 +44,28 @@ test(`HTML link transformation regex matches links`, async () => {
 
   expect(matches.length).toBe(2)
 })
+
+test(`Search and replace node strings using regex matches`, async () => {
+  const nodeString = `Some stuff in a random string
+
+  A new line with some stuff!
+
+  We need to test some <a href=\\"https://old-site.com/hi\\" />link</a> as well!`
+
+  const result = searchAndReplaceNodeStrings({
+    nodeString,
+    node: { __typename: "FakeTypeName", id: "cG9zdDo0OQ==" },
+    pluginOptions: {
+      searchAndReplace: [
+        { search: "(S|s)ome stuff", replace: "some other thing" },
```

Comment:
I had to change it to this to get the tests to pass:
```suggestion
        { search: /(S|s)ome stuff/gm, replace: `some other thing` },
```

But after that it passes :)